src/static/js/getChatUpdate.js

```diff
@@ -0,0 +1,123 @@
+// Simulated frontend data, replace with actual implementation
+let currentTokenId = 0; // Assume the frontend starts with token ID 0
+const name = document.getElementById("name").value;
+console.log(name);
+
+const fakeMessage = [
+  [{ tokenId: 1, updateOn: null, content: "Hello, World!", sender: "Alice" }],
+  [{ tokenId: 2, updateOn: 1, content: "Updated message!", sender: "Alice" }],
+  [
+    { tokenId: 3, updateOn: 2, content: null, sender: "Alice" }, // Deleted message
+  ],
+  [{ tokenId: 4, updateOn: null, content: "New message!", sender: "Bob" }],
+  [{ tokenId: 5, updateOn: 4, content: "Updated message!", sender: "Bob" }],
+  [{ tokenId: 6, updateOn: null, content: "New Message 1", sender: "tester" }],
+  [{ tokenId: 7, updateOn: null, content: "New Message 2", sender: "Bob" }],
+  [{ tokenId: 8, updateOn: null, content: "New Message 3", sender: "Bob" }],
+  [{ tokenId: 9, updateOn: null, content: "New Message 4", sender: "Bob" }],
+  [{ tokenId: 10, updateOn: null, content: "New Message 5", sender: "Bob" }],
+  [{ tokenId: 11, updateOn: null, content: "New Message 6", sender: "Bob" }],
+  [{ tokenId: 12, updateOn: null, content: "New Message 7", sender: "Bob" }],
+  [
+    { tokenId: 13, updateOn: 12, content: null, sender: "Bob" }, // Deleted message
+  ],
+  [{ tokenId: 14, updateOn: null, content: "New Message 8", sender: "Bob" }],
+  [{ tokenId: 15, updateOn: null, content: "New Message 9", sender: "Bob" }],
+  [{ tokenId: 16, updateOn: null, content: "New Message 10", sender: "Bob" }],
+  [{ tokenId: 17, updateOn: null, content: "New Message 11", sender: "Bob" }],
+  [{ tokenId: 18, updateOn: null, content: "New Message 12", sender: "Bob" }],
+  [{ tokenId: 19, updateOn: null, content: "New Message 13", sender: "Bob" }],
+  [{ tokenId: 20, updateOn: null, content: "New Message 14", sender: "Bob" }],
+  [{ tokenId: 21, updateOn: null, content: "New Message 15", sender: "Bob" }],
+  [{ tokenId: 22, updateOn: null, content: "New Message 16", sender: "Bob" }],
+  [{ tokenId: 23, updateOn: null, content: "New Message 17", sender: "Bob" }],
+  [{ tokenId: 24, updateOn: null, content: "New Message 18", sender: "Bob" }],
+  [{ tokenId: 25, updateOn: null, content: "New Message 19", sender: "Bob" }],
+  [{ tokenId: 26, updateOn: null, content: "New Message 20", sender: "Bob" }],
+];
+
+function scrollToBottom() {
+  var element = document.getElementById("messageContainer");
+  element.scrollTop = element.scrollHeight - element.clientHeight;
+}
+
+function getMessages(tokenId) {
+  try {
+    const message = fakeMessage[tokenId];
+    return message;
+  } catch (err) {
+    console.log(err);
+    return null;
+  }
+}
+
+// Function to simulate receiving messages from the backend
+function updateView(tokenId) {
```

Comment:
This `update_view()` function assumes that users will be able to edit messages sent via a live chat. I don't know if that's what we want to implement since I'm not sure whether most live chat features allow you to edit a message that you sent. We should look into this though.

Reply:
We have discussed this already. We think we will reserve edit for later PRs; Please pay attention that the updateView here is purely for updating the display of the page to reflect new messages, and doesn't involve editing messages

Reply:
I'm not even sure we need an edit for live chat messages. I'm not sure that functionality exists for a lot of live chat services. For example, in OverWatch 2, once you send a chat message, the user cannot edit it. So I'm not sure if our live chat should even have an edit feature.

Reply:
Yeah that's true, and that's why we have actually make this probably that last feature we will implement for chat; for now, we can just ignore edit

Reply:
Sounds good to me!